src/SelectState.js

```diff
@@ -44,6 +44,7 @@ class Select extends Component {
         identifier={this.props.identifier}
         label={this.props.label}
         options={this.state.options}
+        selected={this.state.selected || []}
```

Comment:
This component is for specific use of state in react, the other component is for redux store